bin/test_integration

```diff
@@ -32,7 +32,7 @@
 
 if [ -z "$1" ]; then
   echo -e "\n Integration Testing ALL CPI methods"
-  ginkgo -r -p -v --noisyPendings integration
+  ginkgo -r -p -v --noisyPendings integration --skipPackage=concurrency_sqlite
```

Comment:
So you are skipping the concurrency SQLite tests... can you explain? Was these flaky?